### PR TITLE
Add divisus.py

### PR DESCRIPTION
**Pull request by TechPowerAwaits**
This is the initial version of the Python script. The CSV-formatted
output it generates might be based on an invalid understanding of the
structure of CSV files, but the script's core functionality does work.

### CODE REVIEW

**Comment by TechPowerAwaits** [comment]
@108Mbps this is waiting on the approval of PR #1 and then it will have to be rebased.

**Review by 108Mbps — APPROVED** [review]
This looks like some good Python programming.

**Comment by TechPowerAwaits** [comment]
@108Mbps review is stale.

**Pull request merged**
